src/Service/AuthorizationService.php

```diff
@@ -117,33 +117,33 @@ public function isGranted($identity, $permission, $context = null)
         }
 
         if ($this->hasAssertion($permission)) {
-            return $this->assert($this->assertions[(string) $permission], $identity, $context);
+            return $this->assert($this->assertions[(string) $permission], $permission, $identity, $context);
```

Comment:
Actually we have this interface (https://github.com/ZF-Commons/zfc-rbac/blob/develop/src/Assertion/AssertionInterface.php) and it does not implement __toString.

Maybe we should remove this interface completely?